OpenRA.Game.exe.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>

```

Comment:
I was under the impression that this was only a problem for shipped installs, but I may be wrong.